CHANGELOG.md

```diff
@@ -20,6 +20,29 @@ All notable changes to this project will be documented in this file, in reverse
 
 - Nothing.
 
+## 3.0.0 - 2016-12-??
+
+### Added
+
+- [#23](https://github.com/zendframework/zend-expressive-helpers/pull/23) adds support to UrlHelper for generating
+  default route w/o parameters
+- [#9](https://github.com/zendframework/zend-expressive-helpers/pull/9) UrlHelper passes `$routerOptions` to underlying
+  router, if provided.
+- [#27](https://github.com/zendframework/zend-expressive-helpers/pull/27) adds query params and fragment identifier
+   support to `UrlHelper`
```

Comment:
Added some notes on that. Also set an optimistic release date on the 3.0.0 entry. :)